lib/pipe/testomatio.js

```diff
@@ -95,7 +95,7 @@ class TestomatioPipe {
       const q = generateFilterRequestParams({
         type,
         id,
-        apiKey: this.apiKey.trim(),
+        api_key: this.apiKey.trim(),
```

Comment:
I think this is incorrect,

apiKey is used inside the called function https://github.com/testomatio/reporter/blob/master/lib/utils/pipe_utils.js#L30

Reply:
thanks, looks like I misinterpreted it